contract/models/account_analytic_invoice_line.py

```diff
@@ -16,23 +16,44 @@ class AccountAnalyticInvoiceLine(models.Model):
     _name = 'account.analytic.invoice.line'
 
     product_id = fields.Many2one(
-        'product.product', string='Product', required=True)
+        'product.product',
```

Comment:
Reviewers - sorry I killed the diff here - I was having a hard time auditing the fields for required, etc. so had to clean up the spacing